lib/measures/tbd/measure.rb

```diff
@@ -113,8 +113,9 @@ def run(model, runner, user_arguments)
       gen_kiva = true
       model.getSurfaces.each do |s|
         next unless s.isGroundSurface
-        s.outsideBoundaryCondition == "Foundation"
```

Comment:
@rd2 this was checking equality and not setting the outside boundary condition

Reply:
@macumber : thanks! pretty silly really.